src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -979,6 +980,12 @@ void MainWindow::add_render_tab(const QString& label)
             *m_project_manager.get_project(),
             m_ocio_config);
 
+    // Set material drop handler
+    render_tab->set_material_drop_handler(
```

Comment:
It would be cleaner if this was done inside `RenderTab`'s constructor, like it's done for all other handlers. Pass the rendering manager if needed.